app/src/main/res/values/strings.xml

```diff
@@ -16,7 +16,7 @@
     <string name="display_travel_time_minutes">%1$dmin</string>
     <string name="display_travel_time_hours_minutes">%1$dh %2$dmin</string>
     <string name="display_travel_time_days_hours_minutes">%1$dd %2$dh %3$dmin</string>
-    <string name="display_average_speed">%1$.2f km/h</string>
+    <string name="display_average_speed">%1$.2f kph</string>
```

Comment:
Since km/h is the SI unit, I'd prefer that in use.

https://en.wikipedia.org/wiki/Kilometres_per_hour